resources/ext/cli/config-migration

```diff
@@ -99,6 +99,7 @@ function replaceline {
   tmp="$3.tmp.$(date +%s)"
   sed "s/$1/$(echo "$2" | sed -e 's/[\/&]/\\&/g')/g" "$3" > "$tmp"
   mv "$tmp" "$3"
+  chmod 644 "$3"
```

Comment:
Hmm, perhaps depends on the broader context, i.e. if replaceline is used on a sensitive file, or one with special permissions, this won't preserve them, and it could also leave the content more vulnerable than 664 until the chmod.

For a moment I also wondered about the embedded sed expansion, but perhaps that's fine.  Though since this is  bash, suppose it might be possible to use a `${parameter//pattern/string}` expansion instead.

Also, don't know if it matters in the current context, but on principle, I'd use mktemp (or something as safe) to create the temp file, i.e. `tmpfile="$(mktemp "foo-XXXXXXX")"`.  mktemp is fairly portable, though if say Linux vs BSD matters, we'll need to make sure we use common args.